### PR TITLE
feat: editor katext extension

### PR DESCRIPTION
**Pull request by gengxiaoxiaoxin**
The katex extension for richtext-editor is coming. 🎇
Consistent editing experience with the [richtext-editor](https://github.com/halo-sigs/richtext-editor). 

There are currently some known minor issues.
- The bubble menu should not appear when clicking on a formula.
- The style of the formula component is not attractive enough.
- Internationalization support.
- ~~Block-katex paste support.(Now always taken over by inline-paste-Regex.)~~

Anyway, it's time to experience the formula plugin!

https://github.com/halo-sigs/plugin-katex/assets/29268441/ae8bec54-0d42-4b45-8a27-60077dd6a991

```release-note
Add KaTex supports for Editor
```

### CODE REVIEW

**Comment by gengxiaoxiaoxin** [comment]
whether to merge or not requires everyone's vote.

**Review by ruibaby — COMMENTED** [review]
/lgtm

image

**Comment by ruibaby** [comment]
@halo-sigs/halo cc

File: console/package.json [hunk 1]
```diff
@@ -13,7 +13,10 @@
   "dependencies": {
     "@halo-dev/components": "^1.5.0",
     "@halo-dev/console-shared": "^2.6.0",
```
Comment (LIlGG) [inline console/package.json:15]:
```suggestion
    "@halo-dev/console-shared": "^2.7.0",
```

**Review by LIlGG — CHANGES_REQUESTED** [review]

**Pull request merged**